src/vs/base/parts/quickopen/common/quickOpen.ts

```diff
@@ -43,7 +43,8 @@ export interface IAutoFocus {
 
 export enum Mode {
 	PREVIEW,
-	OPEN
+	OPEN,
+	OPEN_BEHIND
```

Comment:
I almost chose that name, but I felt it was like a background job.  Will change.